09a_Custom_Coadd.ipynb

```diff
@@ -0,0 +1,1386 @@
+   "execution_count": null,
+   "id": "c9431dfe-de6f-45f1-b083-09a3b83de627",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "outputRun = f\"{my_outputCollection}/TestWindow1_coadd\"\n",
+    "\n",
+    "tmpButler = Butler(config, writeable=True)\n",
+    "\n",
+    "tmpButler.registry.registerCollection(outputRun, CollectionType.RUN)\n",
+    "print('Made CollectionType.RUN: ', outputRun)\n",
+    "\n",
+    "tmpButler.registry.registerCollection(my_outputCollection, CollectionType.CHAINED)\n",
+    "print('Made CollectionType.CHAINED: ', my_outputCollection)\n",
+    "\n",
+    "collections = [collection]\n",
+    "collections.insert(0, outputRun)\n",
+    "\n",
+    "tmpButler.registry.setCollectionChain(my_outputCollection, collections)\n",
+    "\n",
+    "simpleButler = Butler(butler=tmpButler, collections=[my_outputCollection], run=outputRun)\n",
+    "\n",
+    "del tmpButler"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "33abd84d-f4d5-4dbf-8feb-15308c42c773",
+   "metadata": {},
+   "source": [
+    "Check that the subdirectory of the newly created output collection is first in the list.\n",
+    "The output will go into that specific RUN collection."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "46962fa6-ef50-4933-9d5d-7f66c22a23c8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "simpleButler.registry.getCollectionChain(my_outputCollection)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cec8f5a4-74bf-449b-9246-7e549dc56efe",
+   "metadata": {},
+   "source": [
+    "**Option:** check the output collection run names that currently exist."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a40c9fa9-8955-4cee-bf3b-360bcf07e5ef",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# for c in sorted(simpleButler.registry.queryCollections()):\n",
+    "#     if c.find(my_outputCollection) > -1:\n",
+    "#         print('Found: ', c)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e1740781-d79a-4202-971b-f9ef8f6a9329",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## 3.3. Create a pipeline with the makeWarp and assembleCoadd tasks\n",
+    "\n",
+    "In order to combine the identified visits into a new Coadd, two steps of the larger Data Release Pipeline (DRP) must be included: `makeWarp` and `assembleCoadd`. \n",
+    "Although warped images are created during processing, they are not stored long-term because they take up a lot of space and because they can be easily recreated when needed.\n",
+    "\n",
+    "For more information, review the relevant documentation about <a href=\"https://pipelines.lsst.io/v/weekly/modules/lsst.pipe.base/creating-a-pipeline.html\">creating a pipeline</a> or the <a href=\"https://pipelines.lsst.io/modules/lsst.pipe.tasks/tasks/lsst.pipe.tasks.assembleCoadd.AssembleCoaddTask.html\">assembleCoadd<a> task.\n",
+    "\n",
+    "**The `from_uri` method.**\n",
```

Comment:
Added my best shot: "_A URI is a uniform resource identifier, and as seen below it can take both a file path and additional information: in this case, the additional information is the processing steps that we wish to use, which are defined in the file._" If you know, please suggest what to put? If that's not enough I'll need to ask a pipelines person.